doc.go

```diff
@@ -716,6 +717,19 @@ Format string of error messages shown in the bottom message line.
 If the format string contains the characters `%s`, it is interpreted as a format string for `fmt.Sprintf`. Such a string should end with the terminal reset sequence.
 For example, "\033[4m%s\033[0m" has the same effect as "\033[4m".
 
+	fileinfofmt    string    (default "\033[36m%p\033[0m %c %u %g %s %t %L")
+
+Format string of the file info shown in the bottom left corner.
+`%p` is replaced with the file permissions.
+`%c` is replaced with the link count.
+`%u` is replaced with the user.
+`%g` is replaced with the group.
+`%s` is replaced with the file size.
+`%t` is replaced with the last modified time.
+`%l` is replaced with the link target if it exists, otherwise a blank string.
+`%L` is the same as `%l` but with an arrow `-> ` prepended.
+On Windows, the link count, user and group fields are not supported and will be replaced with a blank string if specified. The default for Windows is "\033[36m%p\033[0m %s %t %L".
```

Comment:
The formatting is messed up in docstring.go and the manpage. I'm not an expert on godoc, but you could try:

- Making the "On windows" sentence its own paragraph.
- Maybe adding `-` before the list items would help?

Reply:
I ended up making the "On Windows" part a separate paragraph, but I don't think there's a good way to format lists so that it will work in all the different formats.

I think that trying to support documentation in multiple formats makes the formatting very restrictive, but I would rather discuss it outside of this PR.

Reply:
The following seems to work OK for me. There are some minor issues if the man page is viewed on a narrow screen, but I think we can ignore them. WDYT?

```
Format string of the file info shown in the bottom left corner.
The following substitutions are performed:

    `%p` is replaced with the file permissions.
    `%c` is replaced with the link count.
    `%u` is replaced with the user.
    `%g` is replaced with the group.
    `%s` is replaced with the file size.
    `%t` is replaced with the last modified time.
    `%l` is replaced with the link target if it exists, otherwise a blank string.
    `%L` is the same as `%l` but with an arrow `-> ` prepended.

On Windows, the link count, user and group fields are not supported and will be replaced with a blank string if specified. The default for Windows is "\033[36m%p\033[0m %s %t %L".
```

Reply:
I did notice that indenting text turns it into a code block - I wasn't sure if it would be OK or not, but on second thoughts it's probably fine.

Thanks for your suggestion.